### PR TITLE
Added Via support to spaceman 2% milk

### PR DESCRIPTION
**Pull request by wongyuhao**
## Description

Created spaceman/2_milk.json 
## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/9484
## Checklist

- [x ] The QMK source code follows the guide here: https://caniusevia.com/docs/configuring_qmk
- [x ] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x ] I have tested this keyboard definition using VIA's "Design" tab.
- [x ] I have tested this keyboard definition with firmware on a device.
- [x ] I have assigned alpha keys and modifier keys with the correct colors.
- [x ] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Spaceman** [comment]
Has been merged into qmk/master https://github.com/qmk/qmk_firmware/pull/9484

**Comment by Spaceman** [comment]
Also the vendor should be changed to 5342

File: src/spaceman/2_milk.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "spaceman_2_milk",
+  "vendorId": "0x5350",
```
Comment (epid) [inline src/spaceman/2_milk.json:3]: Should this be 0x5342 ?

Comment (Spaceman) [reply src/spaceman/2_milk.json:3]: yes

**Comment by Spaceman** [comment]
Nudging @wongyuhao

**Comment by wongyuhao** [comment]
Apologies for the delay, I've changed it.

**Comment by Spaceman** [comment]
@Wilba6582 this is no longer pending qmk merge

**Pull request merged**